Sources/YSnackbar/Views/SnackContainerView.swift

```diff
@@ -33,11 +33,22 @@ internal class SnackContainerView: UIView {
 
     internal var appearance: SnackbarManager.Appearance
 
-    private(set) var hostViews: [SnackHostView] = []
+    internal var hostViews: [SnackHostView] = []
 
     private var dampingRatio: CGFloat { 0.6 }
     private var velocity: CGFloat { 0.4 }
-    private var rearrangeAnimationDuration: CGFloat { 0.4 }
+    internal var rearrangeAnimationDuration: CGFloat { 0.4 }
+
+    /// Override for isReduceMotionEnabled. Default is `nil`.
+    ///
+    /// For unit testing. When non-`nil` it will be returned instead of
+    /// `UIAccessibility.isReduceMotionEnabled`,
+    internal var reduceMotionOverride: Bool?
+
+    /// Accessibility reduce motion is enabled or not.
+    internal var isReduceMotionEnabled: Bool {
```

Comment:
I like this pattern, but if we are putting this in many different libraries, I wonder if dependency injection might be something to investigate.